projects/js-packages/components/components/pricing-table/index.tsx

```diff
@@ -50,20 +54,19 @@ export const PricingTableItem: React.FC< PricingTableItemProps > = ( {
 	tooltipTitle,
 } ) => {
 	const [ isLg ] = useBreakpointMatch( 'lg' );
-	const items = useContext( PricingTableContext );
-	const rowLabel = items[ index ].name;
-	const defaultTooltipInfo = items[ index ].tooltipInfo;
-	const defaultTooltipTitle = items[ index ].tooltipTitle;
-	const includedLabel = __( 'Included', 'jetpack' );
-	const notIncludedLabel = __( 'Not included', 'jetpack' );
+	const item = useContext( PricingTableContext )[ index ];
+
+	const featureNameLabel = item.name;
+
+	const defaultTooltipInfo = item.tooltipInfo;
+	const defaultTooltipTitle = item.tooltipTitle;
 	const showTooltip = tooltipInfo || ( ! isLg && defaultTooltipInfo );
 
-	let defaultLabel = isIncluded ? includedLabel : notIncludedLabel;
-	defaultLabel = isLg ? defaultLabel : rowLabel;
+	const includedLabel = isIncluded ? INCLUDED_TEXT : NOT_INCLUDED_TEXT;
 
-	if ( ! isLg && ! isIncluded && label === null ) {
-		return null;
-	}
+	const defaultLabel = isLg
+		? includedLabel
+		: featureNameLabel + ( isIncluded ? '' : ` ${ NOT_INCLUDED_ALT_TEXT }` );
```

Comment:
Could we make this translatable with a sprintf and a translator comment? It may be easier for translators that way, and will handle diferent uses of parenthesis.

Reply:
Good idea, done!